### PR TITLE
Fix: Incremental Cursor Field

### PR DESCRIPTION
**Pull request by hash-data**
# Problem
(write a short description here or paste a link to JIRA)
Currently, only indexes go as cursor fields, now onwards all fields will go as incremental fields.

# Solution
(write a short description here or paste a link to JIRA)

# QA steps
 - [ ] automated tests passing
 - [ ] manual QA steps passing (list below)

 
# Risks

# Rollback steps
 - Revert this branch

### CODE REVIEW

**Comment by hash-data** [comment]
Alternate to it is to ask client to make indexes on fields which they want as cursor field.

File: dz_mongodb/db_utils.py [hunk 1]
```diff
@@ -227,4 +210,9 @@ def produce_collection_schema(collection: Collection) -> Dict:
 
             db_schema['schema']['properties'][key] = {"type": val_type}
 
+    # exposing all fields to be incremental
```
Comment (shubham19may) [inline dz_mongodb/db_utils.py:213]: did we check existing source? what happens when we try to modify it?

Comment (shubham19may) [reply dz_mongodb/db_utils.py:213]: basically if the key that was already there shouldn't change

Comment (hash-data) [reply dz_mongodb/db_utils.py:213]: this is only being used to select incremental cursor and in existing source no one connected incremental.

**Review by shubham19may — APPROVED** [review]

**Review by shubham19may — APPROVED** [review]

**Pull request merged**